deploy/docs/Non_Helm_Installation.md

```diff
@@ -60,273 +39,149 @@ First, you'll need to set up the relevant [fields](https://help.sumologic.com/Ma
-sed 's/cluster kubernetes/cluster <CLUSTER_NAME>/g' | \
-kubectl -n sumologic apply -f -
-```
-
-### Customize configuration
-
-If you need to customize the configuration there are two commands to run. First, get the `fluentd-sumologic.yaml` manifest with following command:
-
-```sh
-curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v1.0/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
-sed 's/\$NAMESPACE'"/sumologic/g" >> fluentd-sumologic.yaml
-```
+The following parameter is optional, but we recommend setting it.
+* __sumologic.clusterName__ - An identifier for your Kubernetes cluster.  This is the name you will see for the cluster in Sumo Logic. Default is `kubernetes`.
 
-Next, customize the provided YAML file. Our [plugin](../../fluent-plugin-events/README.md#fluent-plugin-events) allows you to configure fields for events. Once done run the following command to apply the `fluentd-sumologic.yaml` manifest.
+First you will generate the YAML to apply to your cluster.  The following command contains the minimum parameters that can generate the YAML to setup Sumo Logic's Kubernetes collection. This command will generate the YAML and pipe it a file called `sumologic.yaml`. Please note that `--namespace` is required
 
-```sh
-kubectl -n sumologic apply -f fluentd-sumologic.yaml
+```bash
+kubectl run tools \
+  -it --quiet --rm \
+  --restart=Never \
+  --image sumologic/kubernetes-tools:1.0.0-rc.0 -- \
+  template \
+  --name-template 'collection' \
+  --set sumologic.accessId='<ACCESS_ID>' \
+  --set sumologic.accessKey='<ACCESS_KEY>' \
+  --set sumologic.clusterName='<CLUSTER_NAME>' \
+  | tee sumologic.yaml
 ```
 
-The manifest will create the Kubernetes resources required by Fluentd.
-
-
-### Verify the pods are running
+Next, you will need to apply the required CRD's for the Prometheus Operator. This is required before applying the generated YAML.
 
-```sh
-kubectl -n sumologic get pod
+```bash
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
 ```
```

Comment:
We spot some issue for at least `1.14` k8s version using this commands:
```
➜  ~ kubectl -n sumologic apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
error: error validating "https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml": error validating data: ValidationError(CustomResourceDefinition.spec): unknown field "preserveUnknownFields" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceDefinitionSpec; if you choose to ignore these errors, turn validation off with --validate=false
➜  ~ curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml | grep -v preserveUnknownFields | kubectl -n sumologic apply -f -
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
customresourcedefinition.apiextensions.k8s.io/alertmanagers.monitoring.coreos.com configured
```

Reply:
I updated the command to use --validate=false.